src/components/grapher/GrapherTool.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <g class="grapher-tool--tool-dots-container">
+    <circle
+      v-for="dot in grapherToolDots"
+      :key="`${dot.x}-${dot.y}-tool--dot`"
+      class="grapher-tool--dot"
+      :cx="dot.x"
+      :cy="dot.y"
+      r="0.7"
+      data-test="grapher-tool--dot"
+    />
+    <rect
+      v-if="showGrapherToolBox"
+      class="grapher-tool--box"
+      :cx="grapherToolBoxX"
+      :cy="grapherToolBoxY"
```

Comment:
Does `cx` and `cy` actually exist on `rect`? Maybe remove this change from this PR and leave it for the select tool PR.

Reply:
Will do.